### PR TITLE
Trigger build that points to the CLI Stability branch

### PR DESCRIPTION
**Pull request by CartBlanche**

### CODE REVIEW

File: VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs [hunk 1]
```diff
@@ -90,11 +90,11 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
         }
         #endregion
 
-        private void OnMeadowDeviceListCombo(object sender, EventArgs e)
+        private async void OnMeadowDeviceListCombo(object sender, EventArgs e)
         {
             if (e is OleMenuCmdEventArgs eventArgs)
             {
-                var portList = MeadowDeviceManager.GetSerialPorts();
+                var portList = MeadowDeviceManager.GetSerialPorts().Result;
```
Comment (adrianstevens) [inline VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs:97]: any reason not to await this call?

File: VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs [hunk 2]
```diff
@@ -167,7 +167,7 @@ private void OnMeadowDeviceListComboGetList(object sender, EventArgs e)
                 }
                 else if (vOut != IntPtr.Zero)
                 {
-                    var portList = MeadowDeviceManager.GetSerialPorts();
+                    var portList = MeadowDeviceManager.GetSerialPorts().Result;
```
Comment (adrianstevens) [inline VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs:170]: and here ... should we await?

**Review by adrianstevens — APPROVED** [review]

File: VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs (hunk 2 above)
Comment (CartBlanche) [reply VS_Meadow_Extension/VS_Meadow_Extension.Shared/MeadowPackage.cs:170]: @adrianstevens as per above.

**Comment by CartBlanche** [comment]
@adrianstevens interestingly, after the rebase with all the other changes, everything seems to work without the `.Result`. So new push removes the last commit and leave the `await` calls in. Looks a little bit snappier to me too. Let me know.

**Pull request merged**